Sprint-3/slideshow/style.css

```diff
@@ -1 +1,7 @@
 /** Write your CSS in here **/
+
+img {
+    width: 340px;
```

Comment:
why is the height auto, but the width is fixed?

Reply:
just to make sure the height adjusts based on the width.